library/local_ec2_vpc_peering_facts.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
```

Comment:
make sure those modules is not included in the ansible version used in this project - review all modules